pkg/controller/apiclient/mem/apiclient.go

```diff
@@ -44,6 +44,8 @@ func NewAPIClient() apiclient.APIClient {
 func NewPopulatedAPIClient(
 	brokers map[string]*servicecatalog.Broker,
 	serviceClasses map[string]*servicecatalog.ServiceClass,
+	instances map[string]apiclient.InstanceClient,
+	bindings map[string]apiclient.BindingClient,
```

Comment:
Nit: calling these arguments `instanceClients` and `bindingClients` would improve the clarity here, since that's what is actually being passed in.